packages/sit-onyx/src/components/OnyxListbox/OnyxListbox.vue

```diff
@@ -128,13 +208,29 @@ const {
       padding: 0;
     }
 
-    .onyx-listbox-option {
-      height: var(--option-height);
-    }
-
     &:has(&__options:focus-visible) {
       outline: 0.25rem solid var(--onyx-color-base-primary-200);
     }
+
+    &__slot {
+      padding: 0 $wrapper-padding;
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      justify-content: center;
+
+      &--height {
```

Comment:
modifiers should have a semantic name what is going on/more about state variations
maybe 
```suggestion
      &--loading {
```

or if you want to use it on emtpy etc too
```suggestion
      &--high {
```